hack/dev/kind/gitea/deploy.py

```diff
@@ -20,6 +20,7 @@
 GITEA_REPO_NAME_E2E = os.environ.get("GITEA_REPO_NAME", "pac-e2e")
 GITEA_REPO_NAME_PERSO = os.environ.get("GITEA_REPO_NAME_PERSO", "pac")
 OPENSHIFT_ROUTE_FORCE_HTTP = os.environ.get("OPENSHIFT_ROUTE_FORCE_HTTP", False)
+SYESTEM_NAMESPACE = os.environ.get("SYESTEM_NAMESPACE", "pipelines-as-code")
```

Comment:
why do you set this?

Reply:
default ns is pipelines-as-code when its on KIND/MINIKUBE
and for openshift when we install through operator ns is `openshift-pipelines` so

Reply:
how do we handle things if we want to deploy SMEE on openshift as namespace differ